search-commons/src/main/java/no/unit/nva/search/resource/response/ResourceSearchResponse.java

```diff
@@ -0,0 +1,43 @@
+package no.unit.nva.search.resource.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class ResourceSearchResponse {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("type")
+    private String type;
+
+    @JsonProperty("otherIdentifiers")
+    private OtherIdentifiers otherIdentifiers;
+
+    @JsonProperty("recordMetadata")
+    private RecordMetadata recordMetadata;
+
+    @JsonProperty("mainTitle")
+    private String mainTitle;
+
+    @JsonProperty("abstract")
+    private String mainLanguageAbstract;
+
+    @JsonProperty("description")
+    private String description;
+
+    @JsonProperty("alternativeTitles")
+    private List<String> alternativeTitles;
+
+    @JsonProperty("publicationDate")
+    private String publicationDate;
```

Comment:
@brinxmat This was in the document we looked at together, but not sure if we should have moved it to recordMetadata? Can you double check?

Reply:
I guess it is the `{"year": "1232", "month": "12", "day": "02"}` publication date that is the intention